hugegraph-core/src/main/java/com/baidu/hugegraph/backend/query/Condition.java

```diff
@@ -76,6 +76,10 @@
         TEXT_CONTAINS("textcontains", String.class, String.class, (v1, v2) -> {
             return v1 != null && ((String) v1).contains((String) v2);
         }),
+        TEXT_CONTAINS_ENTIRE("textcontainsentire", String.class, String.class,
```

Comment:
my suggestion is to just enhance GraphIndexTransaction.segmentWords instead of adding textcontainsentire operator, check whether there are multiple special symbols in the text param, and then split it into multiple search terms.
we can support form like `Text.contains("(word)")` and `Text.contains("word1|word2|word3")`